### ISSUE TITLE
Mod Incompatibility with malisiscore-1.12.2-6.4.0

### ISSUE THREAD

**Issue by Thezapman1** [Tommsy64/SmartMovingReloaded#13]
Mod version: **0.0.4**
Forge: 1.12.2 **14.23.5.2807**

https://paste.ubuntu.com/p/mncc5bJyhq/

Something about the wrong mixin version:

Exception in thread "main"   <log4j:Event logger="STDERR" timestamp="1548214014308" level="INFO" thread="main">
    <log4j:Message><![CDATA[[java.lang.ThreadGroup:uncaughtException:1052]: org.spongepowered.asm.launch.MixinInitialisationError: Required mixin config mixins.smartmoving.core.json requires mixin subsystem version 0.7.5]]></log4j:Message>
  </log4j:Event>

You should consider making a discord for mod development because I'd love to pitch in with the community to help.

**Comment by Thezapman1** [comment]
The game immediately crashes upon launch with both only just smart moving reloaded and malisis core, however, SMR is compatible just fine with 70 other mods I have installed and tested (even all at once), the crash does not even leave a crash report, so I copied it raw from the log output.

**Comment by HighwayStar54** [comment]
Yep, same. Posted this a while ago but I got inactive on minecraft so I didn't follow the debugging steps tommy told me to do, sorry ;\

**Comment by plaguewolf** [comment]
mixin incompatibilities are simple enough to solve. the problem is that the first mod with mixin to load in alphabetical order is the version that gets used. to solve this, and make sure that the mod with the latest mixin version is used, simply search your log file for mods that supply mixin, and rename the mod with the highest version to 00_modname or something similar, so that it loads first.
